fpdcc/document-search
# Make Book IntegerRangeFields inclusive on the upper bound

## Overview

Previously, IntegerRangeFields were exclusive on the upper bound, which made for confusing form views (e.g. a Section range displayed as `31-32` would appear in the form with the lower bound `31` and the upper bound `33`). This PR implements a custom `InclusiveIntegerRangeField` that shows an inclusive upper bound in the form display.

## Testing Instructions

* Navigate to http://localhost:8000/books/search
* Find a Book with non-null values for section, township, and range and `View details`
* Click `Edit` to go to the edit view
* Confirm that the range values in the form match the range values that you saw on the detail view
